OpenProblemLibrary/ma117DB/set5/srw2_9_63.pg

```diff
@@ -20,35 +20,37 @@ DOCUMENT();        # This should be the first executable line in the problem.
 
 loadMacros(
   "PGstandard.pl",
-  "PGchoicemacros.pl",
+  "PGML.pl",
+  "MathObjects.pl",
   "PGcourse.pl"
 );
 
-TEXT(beginproblem());
 $showPartialCorrectAnswers = 1;
 
+Context("Numeric");
+
 $a = random(2,9,1);
+$ans1 = Compute("-$a");
+$ans2 = Infinity;
+$ans3 = Formula("sqrt(x)-$a");
+
+BEGIN_PGML
+The function [`f(x)=(x+[$a])^2`] is not one-to-one. Choose a largest possible
```

Comment:
There is a unique largest possible domain containing 100, so how about changing this to "Choose the largest possible domain..."

Reply:
Done.